lib/interfaces/swagger-custom-options.interface.ts

```diff
@@ -1,10 +1,30 @@
+import { FastifySwaggerInitOAuthOptions, FastifySwaggerUiConfigOptions } from "fastify-swagger";
```

Comment:
Hmm, importing a package is something we want to avoid doing, as it will lead to a hard dependency on it. Could we make passable to a generic interface and have the type then properly used?

Reply:
Ah, ok.
Will do!